app/server/tests/get_community.test.js

```diff
@@ -0,0 +1,121 @@
+const endpoint = 'https://hwp-express-api-d63404-dev.apps.silver.devops.gov.bc.ca/api';
```

Comment:
```suggestion
const endpoint = `http://${window._env_.REACT_APP_API_REF}:${window._env_.REACT_APP_API_PORT}/api`;
```

Reply:
Is this test meant to use the openshift url or local dev? The way you are using the env variables her is for local dev.